### DISCUSSION TITLE
I2Cにつながるデバイスをオンボードに

### DISCUSSION THREAD

**Discussion by 610t** [algyan/XIAOGYAN#13] [Ideas]
費用面の問題は大いにあると思いますが、なんらかのI2C用のデバイスをオンボードに載せることはできないでしょうか?
例えば、温度、湿度、気圧が取れるBME280とか、
何らかの加速度計(MPU6886とかSH200QとかLSM6DS3?)とか。

**Comment by matsujirushi** [comment]
提案ありがとうございます！

I2Cデバイスを**オンボードに**用意したい意図、狙いは何でしょうか？？

**Reply by 610t** [reply]
すぐに使えるぐらいですか…
加速度計でLED Matrix使って玉転がしゲームとか、
環境センサー情報をLED Matrixに表示したりとか…

うーん、Grove I2C使って外付けで十分な気がしてきました。
すみません、忘れてください。
